File: 400M Race.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y,z;
        scanf("%d%d%d",&x,&y,&z);
        if(x<y&&x<z)
        printf("Alice\n");
        else if(y<x&&y<z)
        printf("Bob\n");
        else
        printf("Charlie\n");
    }
}

File: 400M Race.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int c=sc.nextInt();
            if(a<b&&a<c)
            System.out.println("ALICE");
            else if(b<a&&b<c)
            System.out.println("BOB");
            else
            System.out.println("CHARLIE");
        }
    }
}